### PR TITLE
change doc schema for graph api

### PR DESCRIPTION
**Pull request by luckygirljlm**
add a column 'activatedOnSharedComputer' to API getOffice365ActivationCounts;
add a column 'sharedComputerActivation' to API getOffice365ActivationsUserCounts

### CODE REVIEW

File: api-reference/beta/resources/office365activationcounts.md [hunk 1]
```diff
@@ -2,15 +2,16 @@
...
-| windows10Mobile   | Int64  | The activation count on Windows 10 mobile. |
+| Property                    | Type   | Description                              |
+| :-------------------------  | :----- | ---------------------------------------- |
+| reportRefreshDate           | Date   | The latest date of the content.          |
+| productType                 | String | The product type such as "Office 365 ProPlus", "Project Client", "Visio Pro for Office 365". |
+| windows                     | Int64  | The activation count on Windows. This number includes every single activation on any Windows machine. |
+| mac                         | Int64  | The activation count on Mac OS.          |
+| android                     | Int64  | The activation count on Android device.  |
+| ios                         | Int64  | The activation count on iOS.             |
+| windows10Mobile             | Int64  | The activation count on Windows 10 mobile. |
```
Comment (Lauragra) [inline api-reference/beta/resources/office365activationcounts.md]: On "a" shared computer.

File: api-reference/beta/resources/office365activationsusercounts.md [hunk 2]
```diff
@@ -2,12 +2,13 @@
...
-| reportRefreshDate | Date   | The latest date of the content.          |
-| productType       | String | The product type such as "Office 365 ProPlus", "Project Client", "Visio Pro for Office 365". |
-| assigned          | Int64  | The number of users have been assigned for the product license. |
-| activated         | Int64  | The number of users have activated the product. |
+| Property                 | Type   | Description                              |
+| :----------------------- | :----- | ---------------------------------------- |
+| reportRefreshDate        | Date   | The latest date of the content.          |
+| productType              | String | The product type such as "Office 365 ProPlus", "Project Client", "Visio Pro for Office 365". |
+| assigned                 | Int64  | The number of users have been assigned for the product license. |
+| activated                | Int64  | The number of users have activated the product. |
```
Comment (Lauragra) [inline api-reference/beta/resources/office365activationsusercounts.md]: On "a" shared computer.

File: api-reference/beta/api/reportroot_getoffice365activationcounts.md [hunk 3]
```diff
@@ -51,6 +51,7 @@ The CSV file has the following headers for columns.
 - Android
 - iOS
 - Windows 10 Mobile
+- Activated On Shared Computer
```
Comment (Lauragra) [inline api-reference/beta/api/reportroot_getoffice365activationcounts.md]: Technically "On" should be lowercase. Is it capped in the header?

File: api-reference/beta/api/reportroot_getoffice365activationcounts.md [hunk 4]
```diff
@@ -99,7 +100,7 @@ Follow the 302 redirection and the CSV file that downloads will have the followi
 HTTP/1.1 200 OK
 Content-Type: application/octet-stream
 
-Report Refresh Date,Product Type,Windows,Mac,Android,iOS,Windows 10 Mobile
+Report Refresh Date,Product Type,Windows,Mac,Android,iOS,Windows 10 Mobile, Activated On Shared Computer
```
Comment (Lauragra) [inline api-reference/beta/api/reportroot_getoffice365activationcounts.md]: Would be good to lowercase "On".

File: api-reference/beta/resources/office365activationsusercounts.md (hunk 2 above)
Comment (luckygirljlm) [reply api-reference/beta/resources/office365activationsusercounts.md]: resolved. :)

**Comment by JohnAustin-MSFT** [comment]
Resource topics for the Reports API lack a summary paragraph. This is a global issue. ~62 topics in beta are affected.  I have created a work time and assigned myself to address the issue in a future PR.

**Pull request merged**